docs/server/source/drivers-clients/http-client-server-api.rst

```diff
@@ -46,8 +46,10 @@ Transactions
 
    Get the transaction with the ID ``tx_id``.
 
-   This endpoint returns a transaction only if a ``VALID`` block on
-   ``bigchain`` exists.
+   This endpoint returns a transaction if it was included in a ``VALID`` block,
+   if it is still waiting to be processed ``BACKLOG`` or is still in an
+   undecided block ``UNDECIDED``. BigchainDB does not keep state about invalid
```

Comment:
I would put parentheses around `BACKLOG` and `UNDECIDED`, otherwise those words break up the flow of the sentence.